fhir-persistence/src/main/java/com/ibm/fhir/persistence/context/FHIRPersistenceContextFactory.java

```diff
@@ -39,16 +40,20 @@ public static FHIRPersistenceContext createPersistenceContext(FHIRPersistenceEve
      * @param includeDeleted flag to tell the persistence layer to include deleted resources in the operation results.
      */
     public static FHIRPersistenceContext createPersistenceContext(FHIRPersistenceEvent event, boolean includeDeleted) {
-        return new FHIRPersistenceContextImpl(event, includeDeleted);
+        return FHIRPersistenceContextImpl.builder(event)
+                .withIncludeDeleted(includeDeleted)
+                .build();
     }
     
     /**
      * Returns a FHIRPersistenceContext that contains a FHIRPersistenceEvent instance.
      * @param event the FHIRPersistenceEvent instance to be contained in the FHIRPersistenceContext instance
-     * @param ifNoneExist flag to tell the persistence layer to apply conditional create-on-update logic.
+     * @param ifNoneMatch flag to tell the persistence layer to apply conditional create-on-update logic.
```

Comment:
wow, good catch

Reply:
yeah...can't tell you how many times I typed exist instead of match :-)